plugins/table/src/ReactDataTable.tsx

```diff
@@ -109,16 +110,16 @@ export default function ReactDataTable(props: DataTableProps) {
   /**
    * Format text for cell value
    */
-  function cellText(key: string, format: string | undefined, val: any) {
-    if (key === '__timestamp') {
+  function cellText(key: string, format: string | undefined, val: DataRecordValue) {
+    if (isTimeColumn(key)) {
       let value = val;
       if (typeof val === 'string') {
         // force UTC time zone if is an ISO timestamp without timezone
         // e.g. "2020-10-12T00:00:00"
         value = val.match(/T(\d{2}:){2}\d{2}$/) ? `${val}Z` : val;
         value = new Date(value);
       }
-      return formatTimestamp(value) as string;
+      return formatTimestamp(value as Date | number | null) as string;
```

Comment:
can you set this above instead of coercing? `let value: Date | number | null = val; ...`

Reply:
Actually tried that, but ran into error when passing it to `new Date`. Obviously `new Date` doesn't accept `null`.